image-meta-tag/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+    name: image-meta-tag
+    version: "0.4"
+
+source:
+    git_url: https://github.com/SciTools-incubator/image-meta-tag
+    git_tag: 0.4.1
+
+build:
+    number: 0
```

Comment:
If the tests continue to fail on Appveyor (which tests the build on OSX), a switch can be put in here to not build on that system:

- skip: True # [osx]

But this may not be necessary unless Appveyor continues to fail today.  Let's see how it goes.